.github/workflows/test-install-script.yml

```diff
@@ -0,0 +1,48 @@
+name: test-install-script
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/test-install-script.yml'
+      - 'wall_e_install.sh'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/test-install-script.yml'
+      - 'wall_e_install.sh'
+
+jobs:
+  linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Ubuntu 20.04
+            host_runner: ubuntu-latest
+            package_manager: apt
+            docker_image: ubuntu:20.04
+            python_package: python2 python3
+            python2_ex: python2
+            python3_ex: python3
+            extra_setup_command: echo "No extra command"
```

Comment:
replicate this for other ubuntu versions

Reply:
We will be installing only the 20.04 version for the attendees, so should I add it for other versions of ubuntu too?

Reply:
Testing should be done for Ubuntu 18 until Ubuntu 22.04